jme3-core/src/main/java/com/jme3/math/FastMath.java

```diff
@@ -843,6 +843,19 @@ public static float determinant(double m00, double m01, double m02,
                 * (m10 * det12 - m11 * det02 + m12 * det01));
     }
 
+    /**
+     * Returns a random float between min and max.
+     *
+     * @param min the desired minimum value
+     * @param max the desired maximum value
+     * @return A random int between <tt>min</tt> (inclusive) to
```

Comment:
Hi John, could you please correct the javadoc?
`@return A random int between...`
should be
`@return A random float between...`